crates/database/src/onchain_broadcasted_orders.rs

```diff
@@ -15,6 +15,21 @@ pub enum OnchainOrderPlacementError {
     Other,
 }
 
+impl OnchainOrderPlacementError {
+    pub fn to_metrics_label(&self) -> &str {
+        match self {
+            Self::QuoteNotFound => "quote not found",
+            Self::InvalidQuote => "invalid quote",
+            Self::PreValidationError => "pre-validation error",
+            Self::DisabledOrderClass => "disabled order class",
+            Self::ValidToTooFarInFuture => "valid to too far in future",
+            Self::InvalidOrderData => "invalid order data",
+            Self::InsufficientFee => "insufficient fee",
+            Self::Other => "unspecified error",
```

Comment:
These long names might be annoying to type in Prometheus metrics, might makes sense to make them more concise and indentifier-like.

100% OK as is though.